backend/inspirehep/matcher/api.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import requests
+from flask import current_app
+from inspire_dojson.utils import get_recid_from_ref, get_record_ref
+from inspire_matcher import match
+from inspire_utils.dedupers import dedupe_list
+from inspire_utils.record import get_value
+
+from .utils import grobid_to_reference
+
+
+def get_reference_from_grobid(query):
+    data = {"citations": query}
+    url = current_app.config["GROBID_URL"]
+    result_xml = requests.post(url, data=data).content
```

Comment:
You should check the HTTP status code here I guess (`response.raise_for_status()`).